### PR TITLE
Update choo

### PR DESCRIPTION
**Pull request by tornqvist**
This is a preemptive PR in anticipation of https://github.com/choojs/choo/issues/700.

### CODE REVIEW

**Comment by shotlom** [comment]
@tornqvist  What's going with the tests? I forked bankai recently from master and just wanted to update the README with some details on documentify, but the tests are not passing. 

How can a readme update fail tests??

**Comment by goto-bus-stop** [comment]
@shotlom the failure on your PR is because of a test bug on windows, don't worry about it

**Comment by shotlom** [comment]
Ok - but does that mean the pull request dies?

**Comment by tornqvist** [comment]
@shotlom Our PRs are failing for different reasons, and it doesn't means that your PR is dead.

**Comment by shotlom** [comment]
Thanks @tornqvist - I don't understand how I can fork the master prod release branch and then have if fail when merging with no conflicts unless prod release is failing. What have I done wrong here?

**Comment by tornqvist** [comment]
This should be ready to merge now that `choo@7` is published.

File: ssr/choo.js [hunk 1]
```diff
@@ -80,8 +76,8 @@ module.exports.render = function (app, route, cb) {
     var res = { state: state }
     res.body = app.toString(route, state)
     delete res.state._experimental_prefetch // State needs to be serializable.
-    if (app.state.title) res.title = app.state.title
-    if (app.state.language) res.language = app.state.language
+    if (state.title) res.title = state.title
```
Comment (goto-bus-stop) [inline ssr/choo.js]: Could we maintain Choo 6 compatibility by doing something like `if (state.title || app.state.title)` here?

Comment (tornqvist) [reply ssr/choo.js]: Yes, that seems very reasonable 👍

**Pull request merged**